tests/dummy/app/templates/modifier.hbs

```diff
@@ -148,21 +150,18 @@
   <section class="vertical-demo">
     <h3>Vertical</h3>
 
-    {{#sortable-group
-            data-test-vertical-demo-group-no-css
-            handleVisualClass=handleVisualClass onChange=(action "update")
-            model=model.items as |group|
-    }}
-      {{#each group.model as |item|}}
-        {{#group.item data-test-vertical-demo-item-no-css tagName="li" model=item  as |groupItem|}}
+    <ol {{sortable-group data-test-vertical-demo-group-no-css
+                         onChange=this.update handleVisualClass=handleVisualClass
+                         groupName="verticle-no-css"}}>
+      {{#each this.model.items as |item|}}
+        <li data-test-scrollable-demo-handle-item-no-css {{sortable-item model=item groupName="verticle-no-css"}}>
           {{item}}
-          {{#groupItem.handle data-test-vertical-demo-handle-no-css class="handle"}}
-            <span data-item={{item}}>
-              <span>&vArr;</span>
-            </span>
-          {{/groupItem.handle}}
-        {{/group.item}}
+          <span class="handle" data-test-vertical-demo-handle-no-css data-item={{item}} {{sortable-handle}}>
```

Comment:
Doesn't `{{sortable-handle}}` also need to know the `groupName` as `sortable-item` and `sortable-group`?

Reply:
{{sortable-handle}} only really matters when you click on it, so when that happens it looks for the {{sortable-item}} its contained in and gets the group from there.

{{sortable-item}} needs to know some stuff about the group to register it at creation time, before its actually put in the DOM, so group has to be passed to it.      There was another version in which {{sortable-item}} looked up {{sortable-group}} at runtime and you didnt have to pass it in, but there were concerns about performance and hence the service registration came about.